src/Symfony/Component/Process/InputStream.php

```diff
@@ -20,12 +20,15 @@
  */
 class InputStream implements \IteratorAggregate
 {
+    /** @var null|callable */
     private $onEmpty = null;
     private $input = array();
     private $open = true;
 
     /**
      * Sets a callback that is called when the write buffer becomes empty.
+     *
+     * @param null|callable $onEmpty
```

Comment:
@nicolas-grekas , what do you think about not adding docblocks for input/output interface if whole of it is described by method header, like here ?
they were mostly useful when php wasn't that typehinted as it is now, with php7 it's possible to get rid of those annotations, as they are duplicating the header (as long as you don't write crazy code that allows 10 different types on input :D)